agent/node.go

```diff
@@ -49,6 +49,10 @@ type NodeConfig struct {
 	// Secret to be used on the first certificate request.
 	Secret string
 
+	// ExternalCAURLs is a list of initial URLs to which a manager node
+	// will make certificate signing requests for node certificates.
+	ExternalCAURLs []string
```

Comment:
Like `Secret` above, this is another way of passing down initial command line options. It starts from  `swarmd/main.go` -> `agent.NewNode()` -> `(* Node).runManager()` -> `manager.New()` -> `(*Manager).Run()` where it is finally used in `store.CreateCluster()` as part of the initial cluster spec.